src/components/donation/Steps/Splits/Splits.tsx

```diff
@@ -58,14 +46,17 @@ export default function Split({
 
   return (
     <div className="grid content-start p-4 @md/steps:p-8">
-      <BackBtn type="button" onClick={() => dispatch(setStep("donate-form"))} />
+      <BackBtn
+        type="button"
+        onClick={() => setState({ ...props, step: "donate-form" })}
+      />
       <h4 className="mt-4">Sustainability Fund Donation</h4>
       <p className="mb-6 text-navy-l1">
         Create a sustainable impact: Give today, give forever
       </p>
 
       <LockedSplitSlider
-        disabled={widgetConfig?.splitDisabled}
+        disabled={props.init.widgetConfig?.splitDisabled}
```

Comment:
still disable this even though this step would actually be skipped on fixed splits